### PR TITLE
Expanded DJ station

### PR DESCRIPTION
**Pull request by Mabblies**
I made the DJ Station's kitchen fully functional, added a security camera console board to the station, added a closet with Russian clothes, and added some extra clothing options to the closet.

### CODE REVIEW

**Comment by FlavoredCactus** [comment]
Screenshots

**Comment by Mabblies** [comment]
http://puu.sh/p71vU/44ae4ee9a3.png
http://puu.sh/p71zc/092276ba04.png

Not pictured is the bedroom, where I added a stack of metal and glass and a security camera board. I also tested and make sure you can see the station cameras using a camera console on the DJ station, and you can. The closets in the kitchen are the 3 closets in the Chef's kitchen.

File: code/game/objects/structures/crates_lockers/closets/gimmick.dm [hunk 1]
```diff
@@ -47,6 +47,22 @@
...
+	new /obj/item/clothing/head/bearpelt (src)
+	new /obj/item/clothing/head/bearpelt (src)
+	new /obj/item/clothing/head/bearpelt (src)
+	new /obj/item/clothing/head/bearpelt (src)
+	new /obj/item/clothing/head/bearpelt (src)
+	new /obj/item/clothing/shoes/steeltoe (src)
+	new /obj/item/clothing/shoes/steeltoe (src)
+	new /obj/item/clothing/shoes/steeltoe (src)
+	new /obj/item/clothing/shoes/steeltoe (src)
+	new /obj/item/clothing/shoes/steeltoe (src)
```
Comment (FlavoredCactus) [inline code/game/objects/structures/crates_lockers/closets/gimmick.dm]: If possible could you undo these changes to the gimmick/russian locker, put a standard grey locker there and place all the Russian stuff you want to put on the same tile as that locker. when the world is created the objects will start inside the new locker. for 2 reasons:

1) the gimmick lockers shouldn't be used in the map.
2) the gimmick lockers share the same icon_state as the time machine and that may confuse some folks in game if someone caries the green locker around.

**Comment by FlavoredCactus** [comment]
bogtvqt
In addition to the random wall on top of the disposal track, the trashbin really shouldnt be where a wall should be being held from the void of space by 1 single window, please find a new place for the trashcan and put a full 4 pane window or wall there

4n9sqvx
this window is on top of a wall

c7cpm9x
these windows are not consistent. you have the 4 pane on grilles on the right and 1 full tile window on the left.

please fix these oddities

**Comment by Mabblies** [comment]
Alrighty, I will fix all that

**Pull request closed without merging**